src/components/v5/common/ActionSidebar/partials/forms/StagedPaymentForm/utils.ts

```diff
@@ -25,7 +25,16 @@ export const getStagedPaymentPayload = (
     colonyAddress: colony.colonyAddress,
     createdInDomain,
     fundFromDomainId: 1,
-    payouts: [],
+    payouts: values.stages.map((stage) => ({
+      recipientAddress: values.recipient,
+      tokenAddress: stage.tokenAddress,
+      amount: stage.amount,
+      claimDelay: '0',
+      tokenDecimals:
+        colonyTokens?.find(
+          ({ token }) => token.tokenAddress === stage.tokenAddress,
+        )?.token.decimals || DEFAULT_TOKEN_DECIMALS,
+    })),
     isStaged: true,
     networkInverseFee,
     annotationMessage: values.description,
```

Comment:
I cannot comment on the exact line below but `stages` array items do not need to contain the `tokenDecimals` property